pgxnclient/commands/__init__.py

```diff
@@ -169,7 +169,7 @@ def __make_subparser(self, parser, subparsers,
 
         glb = subp.add_argument_group(_("global options"))
         glb.add_argument("--mirror", metavar="URL",
-            default = 'http://api.pgxn.org/',
+            default = 'https://api.pgxn.org/',
```

Comment:
Note that this has a) changed the behaviour of the program (but it's better this way) and b) broken the test suite. I've fixed it and merged.

Reply:
Doh! But this shows how pgxnclient could benefit from CI integration. e.g. #34 :)